packages/cms-cli/commands/hubdb.js

```diff
@@ -71,6 +77,43 @@ function configureHubDbCreateCommand(program) {
   addConfigOptions(program);
 }
 
+function configureHubDbImportCommand(program) {
+  program
+    .version(version)
+    .description('Import HubDB table rows')
+    .arguments('<tableId> <src>')
+    .action(async (tableId, src, command = {}) => {
+      setLogLevel(command);
+      logDebugInfo(command);
+      const { config: configPath } = command;
+      loadConfig(configPath);
+      checkAndWarnGitInclusion();
+
+      if (!(validateConfig() && (await validatePortal(command)))) {
+        process.exit(1);
+      }
+      const portalId = getPortalId(command);
+
+      try {
+        const table = await importHubDbTableRows(
+          portalId,
+          tableId,
+          path.resolve(getCwd(), src)
```

Comment:
The pattern we use with the `fetch` command is to make the local destination optional with the cwd as default.
So in the commander pattern you could use `[src]` instead.

Reply:
I think it's possible that the name for this command is misleading. This command is more like the `create` command where it "uploads" a table (or, actually just appends the rows).

Reply:
Maybe it should be `hubdb replace`?

Reply:
I think that with `replace` I would expect metadata and columns would also get affected. If you think that's more valuable, I can look at that.

Otherwise, `append` might be more accurate, but I'd defer to you guys.

Reply:
> Otherwise, append might be more accurate, but I'd defer to you guys.

Does it only add rows or does it also update them? What happens if the schema is different?

Is adding `hs hubdb import` a blocker to finishing up the sync script for the developers site? If not, perhaps we should remove it from this PR since there is some subtlety that needs to be worked out. I bring it up just so that we can keep things moving forward.

Reply:
For the sync script, we need a way to replace the table in its place (as opposed to deleting and creating). `hs import` isn't necessary if there is a way to delete all rows AND update a table.

I think `import` might get conflated with the actual `/import` hubdb endpoint.